base/pkg/entry.jl

```diff
@@ -376,7 +376,11 @@ function update(branch::AbstractString)
                         LibGit2.fetch(repo)
                         LibGit2.merge!(repo, fastforward=true)
                     catch err
-                        show(err)
+                        if isa(err, LibGit2.GitError)
+                            print("Package $pkg cannot be updated. Error: $(err.msg) Class: $(err.class)")
+                        else
+                            show(err)
```

Comment:
We've already been through it. Rethrowing exception would terminate update process, and the rest of the packages would never be updated until this error is resolved. We do not want to do that, because there are many situation when update of package can fail, i.e. you are in a different branch of the package.